docs/ml/upload-model.md

```diff
@@ -8,46 +8,81 @@ aliases:
   - /manage/data/upload-model/
   - /manage/ml/upload-model/
 description: "Upload a Machine Learning model to Viam to use it with the ML Model service."
-# SME: Aaron Casas
+# SME: Steven B.
 ---
 
-On the [**DATA** tab](https://app.viam.com/data/view) in the Viam app, navigate to the **Models** subtab.
+## Upload an existing model to the registry
 
-![Add new model](/ml/add-new-model.png)
+Trained ML models are available in the [Viam registry](https://app.viam.com/registry) for public access.
```

Comment:
Only if you make them accessible! This leaves room for confusion.

Reply:
Removed